### PR TITLE
light/dark theme

### PR DESCRIPTION
**Pull request by alichraghi**

### CODE REVIEW

File: public/CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-hexops.com
```
Comment (slimsag) [inline public/CNAME:1]: don't know why this file was deleted, I restored it just now.

**Review by slimsag — APPROVED** [review]

**Comment by slimsag** [comment]
Looks great, thanks so much!

If you do this for other sites, please commit formatting changes separately if possible (makes it easier to review)

File: public/CNAME (hunk 1 above)
Comment (alichraghi) [reply public/CNAME:1]: sorry i deleted public/ because i though there's a cache problem

**Comment by alichraghi** [comment]
right, will do it in zigmonthly and devlog

**Pull request merged**